code/index.html

```diff
@@ -1,30 +1,136 @@
+                <span>
+                  I agree to the terms and conditions
+                </span>
+              </label>
+              <div class="divider"></div>
+              <label id="submit-label" for="submit">
+                <input id="submit-input" type="submit" value="Transfer">
+              </label>
+            </form>
+          </div>
+
+          <!-- HERE IS THE ACCORDION PART -->
+          <div class="faq-container">
+            <h1>Frekittenly asked questions</h1>
+
+            <div class="accordion-container">
+
+              <div id="accordion-1" class="accordion">
+                <h3 id="accordion-header-1" class="accordion-header">What is Kitten Transfer?</h3>
+                <p class="accordion-content">
+                    Kitten Transfer is a service for kitten owners who enjoy transferring their kitten quickly and at low cost. Kittens of all sizes and shapes can be transferred, just make sure you have a reliable internet connection!
+                </p>
+              </div>
+
+              <div id="accordion-2" class="accordion">
+                <h3 id="accordion-header-2" class="accordion-header">Why would you want to transfer a kitten?</h3>
+                <p class="accordion-content">
+                    Oh a number of reasons! For example, have you ever had a cat on an airplane? Oh those poor things - hungry and afraid! Why put your darling through that. 
+                </p>
+              </div>
+
+              <div id="accordion-3" class="accordion">
+                <h3 id="accordion-header-3" class="accordion-header">Are there any side-effects?</h3>
+                <p class="accordion-content">
+                    We are happy to say that there are no known side-effects to our Kitten Transfers. Years of R&D has really paid off!
+                </p>
+              </div>
+
+              <div id="accordion-4" class="accordion">
+                <h3 id="accordion-header-4" class="accordion-header">Can i transfer my puppy?</h3>
+                <p class="accordion-content">
+                    At the moment no, but our product team is always on the lookout for new features -- so check back in shortly! 
+                </p>
+              </div>
+
+              <div id="accordion-5" class="accordion">
+                <h3 id="accordion-header-5" class="accordion-header">Is insurance included in the price?</h3>
+                <p class="accordion-content">
+                    Meow
+                </p>
+              </div>
+            </div>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+          </div>
+        </div>
+        <div class="bottom-lower">
+          <div class="kt-logo">
+            <p>Thanks for flying with...</p>
+            <p class="logo-text">Kitten Transfer</p>
+          </div>
+        </div>
```

Comment:
This could be a `footer`! [Here](https://medium.com/@zac_heisey/7-alternatives-to-the-div-html-tag-7c888c7b5036)'s some further reference on what we can use to avoid a sea of `div`s (although I'm fully aware we cannot omit them completely).